packages/cli-utils/src/commands/shared/logger.ts

```diff
@@ -54,6 +54,9 @@ export function externalError(message: string, error: unknown) {
     ) {
       title = 'code' in error ? 'System Error' : 'Error';
       text = (error as Error).message.trim();
+    } else if ('stack' in error && typeof error.stack === 'string') {
```

Comment:
Quick note: I'm just committing this, we always need it in development, and there's no harm in seeing more output when someone reports an issue